src/routes/docs/advanced/platform/release-policy/+page.markdoc

```diff
@@ -0,0 +1,122 @@
+
+Currently, the latest stable version of the Appwrite API is `/v1`. We use this prefix in all our API endpoint paths to allow API versioning. Any Appwrite version regardless of its version, is committed to maintain backward compatibility for each version of the API you decided to use as long as this API version is still under maintenance support.
+
+We will provide standard maintenance support for the last 3 API versions. Once a version is no longer in this maintenance period, we will continue to provide support for security fixes for an additional ten years. Based on usage, we may decide to extend support for a specific version.
+
+# SDK versioning {% #sdk-verversioningsions %}
+
+For our different SDKs, we follow the Semantic Versioning (semver) protocol to assign versions to our releases. This means that we assign version numbers using a three-part system: major, minor, and patch. The major version changes when we make significant changes to our API or product, which may require significant changes to developers' code. The minor version changes when we add new features or functionality that do not significantly impact developers' systems. Finally, the patch version changes when we make bug fixes or minor improvements.
+
+All Appwrite SDKs will have backward compatibility with the Appwrite APIs. In case a new version of the API or product has been released, you should expect your applications to continue working properly without any action from your side. Once we release a major version of the SDK and you decide to upgrade, do look in the changelog of the relevant SDK to understand what changes have been made and what adjustments are required.
+
+We provide early notice to developers and slowly introduce breaking changes to let developers adjust their application at a reasonable pace. We will also continue to support the last five versions of each SDK to provide developers with more flexibility and time to adjust their apps to take advantage of new features.
+
+# Self-hosted versioning {% #self-hosted-versioning %}
+
+When you self-host Appwrite, we also follow the Semantic Versioning (semver) protocol for versioning our releases. This means that we assign version numbers using a three-part system: major, minor, and patch. The major version changes when we make significant changes to our API or product, which may require significant changes to developers' apps. The minor version changes when we add new features or functionality that do not significantly impact developers' existing apps. Finally, the patch version changes when we make bug fixes or minor improvements.
+
+All the self-hosted versions of Appwrite `>=1.x.x` continue to have support and backward compatibility with the Appwrite API and SDKs. In case a new version of the products has been released and you decide to update, you should expect your applications to continue working properly without any action from your side.
```

Comment:
This sounds like a big commitment. `>= 1.x.x` is a very wide open limit. 

Firstly I feel its important to clarify between the terms Support and Extended Security Support. 
10 years later if someone comes with a feature request for 1.0.0 what do we do? Looking at the document it is "supported"

So lets first clarify their meaning. What does "support" entail? and what does extended security support entail ?